AdminWebsite/AdminWebsite.UnitTests/Helper/DateListMapperTest.cs

```diff
@@ -31,5 +33,23 @@ public void Should_return_empty_list_if_end_and_start_dates_is_the_same()
 
             result.Count.Should().Be(expectDays);
         }
+        
+        [Test]
+        public void Should_return_range_of_dates_not_included_weekends_and_public_holidays()
+        {
+            var startDate = new DateTime(2020, 10, 1, 4, 30, 0, 0);
```

Comment:
should we have tests for more edge case ranges?
same start date end date, 
start date or end date on weekend
end date larger than start date 💥

Reply:
That already exists in another test :)